metering/metering-using-metering.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 .Prerequisites
 
 * xref:../metering/metering-installing-metering.adoc#metering-install-operator_installing-metering[Install Metering]
-* Review the details about the available options that can be configured for a xref:../metering/metering-about-reports.adoc#metering-about-reports[Report] and how they function.
+* Review the details about the available options that can be configured for a xref:../metering/reports/metering-about-reports.adoc#metering-about-reports[Report] and how they function.
```

Comment:
Why is this a separate assembly?

Reply:
The "Using metering" section walks you through the basics of creating and viewing a report. These are the two main ways a user can interact with metering. The "Reports" section is more referential, containing all the possible things you can do with reports. I still have a few topics to add here in the Reports section.